### ISSUE TITLE
[Blazor] .NET6 Preview 5 - Blazor Server App Published to Azure Web App fails

### ISSUE THREAD

**Issue by ameyasubhedar** [dotnet/aspnetcore#33863]
Early Evaluating .NET 6 - Preview 5

Blazor Server App Published to Azure Web App fails with as error that I couldn't make any sense of.

The same project works on the local machine but fails during startup once published.

Publish Settings:
![image](https://user-images.githubusercontent.com/13166236/123496066-1f066900-d5db-11eb-8ec9-ece50ee10e77.png)

![image](https://user-images.githubusercontent.com/13166236/123495972-aa332f00-d5da-11eb-82b3-c06da4bb0d77.png)

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us.
Can you please try to do a framework dependent deployment instead and see whether that helps?

**Comment by ameyasubhedar** [comment]
If I ignore the warning and force publish, still the same error for both Self-Contained and Framework Dependent.

**Comment by mkArtakMSFT** [comment]
@ameyasubhedar  can you try out the latest 6.0-preview6 release and see whether the issue remains? Thanks!

**Comment by ameyasubhedar** [comment]
@mkArtakMSFT @pranavkm , I have tried with the latest 6.0-preview6 and I can confirm the the Blazor Server App publish issue is fixed. Thanks !

But now in preview-6 I am facing the issue with **Blazor Web Assembly**, the self contained **browser-wasm** fails to publish. Here is the error info

![image](https://user-images.githubusercontent.com/13166236/127700991-0eaa87f6-3a64-4e85-9b11-d610657d04b5.png)

![image](https://user-images.githubusercontent.com/13166236/127701100-aa853be1-99fe-442b-b26c-b8c246d2e18b.png)

what is this workload microsoft-net-sdk-blazorwebassembly-aot ? I have already installed x86 >net 6 SDK.
